backend/private-graph/graph/schema.graphqls

```diff
@@ -494,13 +494,28 @@ type S3File {
 	key: String
 }
 
-type LogLine {
+type Log {
 	timestamp: Timestamp!
 	severityText: SeverityText!
 	body: String!
 	logAttributes: Map!
 }
 
+type LogEdge {
+	cursor: String!
+	node: Log!
+}
+
+type PageInfo {
+	hasNextPage: Boolean!
+	endCursor: String!
```

Comment:
We could have gotten this on the frontend by just looking at the last item in `edges` but it is nice to just have this explicit to reduce the amount of work the frontend has to do.